### PR TITLE
Move prettier and eslint tasks as Github action

### PR DESCRIPTION
**Pull request by ugogo**

### CODE REVIEW

**Comment by gyndav** [comment]
May I chime in and commit in this branch?

**Comment by gyndav** [comment]
I believe you're aware but just in case
```
WARN asset size limit: The following asset(s) exceed the recommended size limit (244 KiB).
WARN This can impact web performance.
WARN Assets:
WARN   vendors~main.2e9c7c00e2bafa1aaf2f.bundle.js (1.62 MiB)
WARN entrypoint size limit: The following entrypoint(s) combined asset size exceeds the recommended limit (244 KiB). This can impact web performance.
WARN Entrypoints:
WARN   main (1.87 MiB)
WARN       runtime~main.2e9c7c00e2bafa1aaf2f.bundle.js
WARN       vendors~main.css
WARN       vendors~main.2e9c7c00e2bafa1aaf2f.bundle.js
WARN       main.css
WARN       main.2e9c7c00e2bafa1aaf2f.bundle.js
WARN
WARN webpack performance recommendations:
WARN You can limit the size of your bundles by using import() or require.ensure to lazy load some parts of your application.
WARN For more info visit https://webpack.js.org/guides/code-splitting/
```

1.87mo 😳

**Comment by ugogo** [comment]
@gyndav I think the 1.87mo are related to Storybook's build, and not the NPM package 😉 
https://bundlephobia.com/result?p=@heetch/flamingo-react@1.0.3

**Comment by gyndav** [comment]
I'll have a look at it ASAP

**Comment by gyndav** [comment]
Flush cache with closing/reopening

**Comment by gyndav** [comment]
Quick update, I'll be able to debug it through Docker with what Zeit Support sent me about their internal config. They don't store logs... 🤦‍♂

**Comment by gyndav** [comment]
Build is now ✅ 
Reference for the fix https://zeit.co/docs/runtimes#official-runtimes/node-js/node-js-version

I enforced latest NodeJS LTS and aligned everything.
Feel free to merge.

File: .github/workflows/main.yml [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+  build:
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v1
+
+    - name: Install dependencies
+      run: |
+        npm install --prefix packages/react
```
Comment (gyndav) [inline .github/workflows/main.yml]: I'd personally recommend aligning and using `yarn` everywhere instead, that will prevent future debugging headaches

In using the root alias, you'd saved multiples lines and do a _one for all_

Comment (ugogo) [reply .github/workflows/main.yml]: Done ✅

**Pull request merged**